spyder/widgets/sourcecode/api/decorations.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could we add this to `sourcecode/__init__.py`? Or maybe in `sourcecode/manager.py`?

I mean, it's a bit strange to have an `api` module that's not exposed in `spyder.api` :-)